src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -51,14 +53,16 @@ public void Add(Dictionary<string, string> dict)
         {
             foreach (var iteration in Benchmark.Iterations)
             {
+                Dictionary<string, string> copyDict = new Dictionary<string, string>(dict);
                 using (iteration.StartMeasurement())
-                {
-                    dict.Add("key1", "string"); dict.Add("key2", "string"); dict.Add("key3", "string");
-                    dict.Add("key4", "string"); dict.Add("key5", "string"); dict.Add("key6", "string");
-                    dict.Add("key7", "string"); dict.Add("key8", "string"); dict.Add("key9", "string");
-                }
-                for (int i = 1; i <= 9; i++)
-                    Assert.True(dict.Remove("key" + i));
+                    for (int i = 0; i <= 20000; i++)
+                    {
+                        copyDict.Add("TestString1." + i, "string"); copyDict.Add("TestString2." + i, "string");
+                        copyDict.Add("TestString3." + i, "string"); copyDict.Add("TestString4." + i, "string");
+                        copyDict.Add("TestString5." + i, "string"); copyDict.Add("TestString6." + i, "string");
+                        copyDict.Add("TestString7." + i, "string"); copyDict.Add("TestString8." + i, "string");
+                        copyDict.Add("TestString9." + i, "string");
```

Comment:
Each of these is now also boxing an integer, calling ToString on that boxed int, and doing a string.Concat.